report-api-faq.md

```diff
@@ -125,73 +99,36 @@ as there are no settlements being made in that environment. The data that
-
-The Ledger ID does not (in general) change, so you may store it in configuration
-in the same way that you store the MSN.
-We may add display of the Ledger ID on
-[portal.vipps.no](https://portal.vipps.no)
-in the future.
-
-## What text is shown for the payouts in my bank?
-
-We send text on this format to all banks:
-`Utb. <settlement_number> Vippsnr <serial_no>`.
-
-Example: `Utb. 2000810 Vippsnr 117703`.
-
-We have no control over, or information about, how banks handle this on their side,
-or how it is displayed in their reports online or in print.
-
-The file format used to transfer information to the bank is the
-[ISO 20022](https://www.iso20022.org)
-PAIN (PAyment INitiation) format, which - among other things - has a limit of 35
-characters for the payment text.
-
-## How can I get the details for a payment I see in my bank?
-
-**Please note:** The payout from Vipps MobilePay to the merchant is
-the balance at the time of the payout. The sum in the payout, which is what
-the merchant sees in the bank, does not contain specific payments -
-it is simply the balance at the time the payout was made.
-
-The payout details can be found with these endpoints, where `{topic}` is `funds`:
-
-* [`GET:/report/v2/ledgers/{ledgerId}/{topic}/dates/{ledgerDate}`][fetch-report-by-date-endpoint]
-* [`GET:/report/v2/ledgers/{ledgerId}/{topic}/feed`][fetch-report-by-feed-endpoint]
+## What information can I get hold of?
 
-Match the settlement ID in the bank with `pspReference`.
+Right now the only data that is available is the same data that is already
+available in the settlement reports on
+[portal.vipps.no](https://portal.vipps.no),
+or on
+[SFTP report (deprecated)](https://developer.vippsmobilepay.com/docs/settlements/sftp-report-service).
 
-The Report API is designed to provide updated data independently of the bank payments.
-There are several reasons for this, including:
+The only difference is that the data can be fetched over a more modern REST API.
+We aim to provide more information through this API in the future.
 
-1. A merchant can have a negative balance and not get settlement payouts for a long time.
-   Periodic checks of settlements would thus not make sense until the merchant has a positive again.
-1. Merchants have different risk profiles (or could have at least). If we deemed that we would hold
-   back some of the balance from payouts to an airline company.
-   Let's say they sell three tickets at 5000 NOK each. We then change their risk profile saying we
-   need to hold back 7500 NOK and then the settlement run. We will then pay out 7500 NOK
-   (if there are no fees). Now: What captures will then be part of that settlement?
-1. We may, in rare cases, manually correct payouts.
+**Important:** It is not yet possible to use the Report API to retrieve data
+for *Vippsnummer* sales units. The Report API can only be used to fetch data for sales units
+that have API access (which *Vippsnummer* sales units do not have.)
```

Comment:
Should we note something here about it not being the case when using accounting partners?

Reply:
Hmm, maybe, but it's mentioned several other places.